### PR TITLE
Add OpenTelemetry-Matlab to registry

### PR DESCRIPTION
**Pull request by duncanpo**
Add OpenTelemetry-Matlab to registry. This is an unofficial implementation of OpenTelemetry in the MATLAB language.

### CODE REVIEW

**Comment by svrnm** [comment]
I stumbled upon this plugin a while ago and I love the idea to have opentelmetry in Matlab, great work @duncanpo! Are you able to sign the CLA such that we can merge your PR?

**Comment by duncanpo** [comment]
> I stumbled upon this plugin a while ago and I love the idea to have opentelmetry in Matlab, great work @duncanpo! Are you able to sign the CLA such that we can merge your PR?

I am currently having the CLA reviewed by our legal team. When we have clearance, we should be able to sign it.

**Comment by svrnm** [comment]
> > I stumbled upon this plugin a while ago and I love the idea to have opentelmetry in Matlab, great work @duncanpo! Are you able to sign the CLA such that we can merge your PR?
> 
> I am currently having the CLA reviewed by our legal team. When we have clearance, we should be able to sign it.

Good to hear, thanks. if you or they have any questions, follow up here and we can help to redirect them to the right people

**Comment by duncanpo** [comment]
@svrnm, I think we are very close to ready to sign the CLA. I hope I will have more updates this week or next.

File: data/registry/otel-matlab.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 title: OpenTelemetry-Matlab
```
Comment (svrnm) [inline data/registry/otel-matlab.yml:1]:
```suggestion
# cSpell:ignore matlab simulink
title: OpenTelemetry-Matlab
```

**Review by svrnm — APPROVED** [review]

**Comment by cartermp** [comment]
@duncanpo to fix the CI error, you'll need to run `npm run fix` on your branch

**Comment by duncanpo** [comment]
> @duncanpo to fix the CI error, you'll need to run `npm run fix` on your branch

This is fixed now.

**Comment by svrnm** [comment]
Thanks @duncanpo! See https://opentelemetry.io/ecosystem/integrations/ -- it's live already!

**Pull request merged**